src/test/sh/org/apache/aurora/e2e/test_end_to_end.sh

```diff
@@ -364,7 +364,6 @@ test_update() {
   # Check that the update ended in ROLLED_FORWARD state.  Assumes the status is the last column.
   assert_update_state_by_id $_jobkey $_update_id 'ROLLED_FORWARD'
 }
-
 test_update_fail() {
```

Comment:
The variable batch is being tested [here](https://github.com/apache/aurora/pull/37/files/49646d0097546cc070fb0a4b3555cca482dceb28#diff-bc249f48d9141c06b10bfd6a9c8c9ed1R980)

If you mean whether I'm verifying if each batch has the correct number of activate instances, then in this case no. It would be a good addition (do we have such a check for static batch updates already?).

Reply:
The second one. I don't believe we have a check for static batch strategy. Will be good to expand on it.

Reply:
Left a TODO on the end to end tests file. If it it's fine with you, I'll get to this in a separate PR.